var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -252,13 +249,13 @@ def test_superlu(self):
         # copy only works for-dist executables to the correct place
         wd = join_path(test_dir, "SuperLU_DIST")
         with working_dir(wd):
-            self.rm("-rf", "superlu_dist")
+            self.cmd["rm"]("-rf", "superlu_dist")
```

Comment:
For commands that are likely to come from outside spack, it is *far* better to prefer to do them in `python` (e.g. use (`shutil.rmtree` or `os.remove`) than to pull them from the shell environment. Can you use `os.remove` here?